### PR TITLE
feat: add app icon

### PR DESCRIPTION
**Pull request by OtterWays**

### CODE REVIEW

**Comment by Jean-BaptisteC** [comment]
App icon is not available in SVG?

File: pubspec.yaml [hunk 1]
```diff
@@ -47,6 +47,7 @@ dependencies:
   go_router: ^13.2.0
   equatable: ^2.0.5
   loading_animation_widget: 1.2.0+4
+  flutter_launcher_icons: ^0.13.1
```
Comment (luifr10) [inline pubspec.yaml:50]: This package seems useful to me when the logos are different between platforms.
If we use the same icons for all platforms, I think we can do without it, and for Android, I also suggest that you replace the "ic_launcher.png" files with the logo, so the settings do not have to be changed.

**Review by luifr10 — CHANGES_REQUESTED** [review]
I have the feeling that above a certain size the logo looks pixelated, the rendering could be better if a svg was used as suggested by @Jean-BaptisteC  (if available of course).

**Comment by Jean-BaptisteC** [comment]
Use SVG on Android add ability to have only one drawable for all dpi size screen. And add ability to support [theming app icon](https://developer.android.com/develop/ui/views/launch/icon_design_adaptive?hl=fr)

**Comment by OtterWays** [comment]
Android and iOS do not support SVG images for launch icons. However, the logo image may not be large enough to work in high resolution

**Comment by Jean-BaptisteC** [comment]
Not support of SVG for launchs icons on Android is a limitation of flutter?

**Comment by OtterWays** [comment]
No, it's an Android limitation. It's possible (and encouraged) to use an SVG image for all icons and images but not for launch icon

**Comment by Jean-BaptisteC** [comment]
The launcher icon is good but we have to use a drawable SVG for the splashscreen, png is very ugly.

**Comment by OtterWays** [comment]
> The launcher icon is good but we have to use a drawable SVG for the splashscreen, png is very ugly.

Yes of course. All other images will be SVG, including the splashscreen

**Pull request merged**